### PR TITLE
PyQt5 fix proposal

### PR DESCRIPTION
**Pull request by stonebig**
for #27

### CODE REVIEW

**Comment by stonebig** [comment]
````
if __name__ == '__main__':
    #m = app.launch(Twente, title='Temperature 1951 - 2014', size=(900, 400))
    m = app.launch(Twente, 'pyqt', title='Temperature 1951 - 2014', size=(900, 400))
    m.bgcolor('#eee')  # more desktop-like
    print(m.proxy._runtime)
    app.run()
````
````
Serving apps at http://localhost:54307/
<flexx.webruntime.qtwebkit.PyQtRuntime object at 0x00000000075CD588>
get Twente-43649b0/
get Twente-43649b0/flexx-ui.css
get Twente-43649b0/flexx.js
get Twente-43649b0/flexx-ui.js
new ws connection /Twente-43649b0/
````

**Comment by stonebig** [comment]
the XUL version as not the <right-Click> update proposal

File: flexx/webruntime/qtwebkit.py [hunk 1]
```diff
@@ -35,8 +39,13 @@
 size = {size}
 pos = {pos}
 
-app = QtGui.QApplication([])
-m = QtWebKit.QWebView(None)
+if not qt == 'pyqt5':
```
Comment (almarklein) [inline flexx/webruntime/qtwebkit.py]: Can you change this to `if qt != 'pyqt5'`, or `if qt == 'pyqt5' and handle pyqt4/pyside in the `else` clause?

File: flexx/webruntime/qtwebkit.py [hunk 2]
```diff
@@ -24,7 +24,11 @@
         from PySide import QtCore, QtGui, QtWebKit
         qt = 'pyside'
     except ImportError:
-        pass
+      try:
```
Comment (almarklein) [inline flexx/webruntime/qtwebkit.py]: Please indent with multiples of four :)

**Comment by almarklein** [comment]
Except the minor style issues this looks good. Thanks a lot!

**Comment by almarklein** [comment]
The fact that you get Xul when no runtime is specified is intended. Xul is the default backend, because Qt Webkit is slower and does not support webgl.

**Comment by stonebig** [comment]
I hoped there was going to be a modern Chromium motor in PyQt5.5.0/Qt5.5/Windows, so  performant.

**Comment by almarklein** [comment]
Well, maybe it is. I'm not really sure about Qt5. But last time I checked at least webgl was not supported.

**Comment by almarklein** [comment]
Thanks for the fix!

**Pull request merged**